packages/next-on-pages/tests/src/buildApplication/processVercelOutput.test.ts

```diff
@@ -478,7 +478,6 @@ describe('processOutputDir', () => {
 		const outputDir = vercelDir;
 
 		expect(readdirSync(outputDir)).toEqual(['index.js', 'nested']);
-		expect(existsSync(outputDir)).toEqual(true);
```

Comment:
I've removed this because it was constantly [failing in the CI](https://github.com/cloudflare/next-on-pages/actions/runs/7520259900/job/20469739658#step:5:50) for some reason 😕

I could not understand why and I could not reproduce this locally 😕
(I _suspect_ this to be a weird github actions/nodejs bug 😕🤷)

I thought of removing this as it is also an unnecessary check, since as you can see from the line above, we already check the content of the directory, so checking for its existence afterwords is really unnecessary, so hopefully this is fine